bandwidth-file-spec.txt

```diff
@@ -208,6 +208,66 @@ It consists of:
 
     This Line has been added in version 1.1.0 of this specification.
 
+  "percent_eligible_relays=" Int NL
+
+    [Zero or one time.]
+
+    The minimum percentage of eligible measured relays with respect
+    to the number of relays in the consensus to include in this file.
+
+    Implementations SHOULD include this KeyValue when
```

Comment:
Don't copy and paste the same text into lots of definitions. It will get out of sync.
Instead, start a new subsection, and put the text at the start of the subsection.